First version for smart contract

nieldeckx opened the pull request:
- Max 100 NFTs
- Token ID 1-100
- Bonding curve: `(x*x)/2` where x = next token ID
- Token owner can burn token

<img width="638" alt="Screenshot 2021-09-04 at 15 12 51" src="https://user-images.githubusercontent.com/5592676/132095609-74a0a0d9-c1ef-4ca1-8e57-f24a2801223b.png">
Total STX would be 169175. Last one would cost 5000 STX.

philipdesmedt commented on clarity/contracts/arkadiko-nft-glasses.clar:
```diff
@@ -0,0 +1,100 @@
+
+;; SIP009: Get the token URI
+(define-read-only (get-token-uri (token-id uint))
+  (ok (some "https://www.arkadiko.finance/nft-glasses/"))
```
This can probably be just a JSON, e.g. arkadiko.finance/nft-glasses.json.

Can you make a setter for this so the admin keys can change the token URI?

philipdesmedt commented on clarity/package.json:
```diff
@@ -0,0 +1,33 @@
+  "name": "arkadiko-nft-glasses",
+  "author": {
+    "name": "Arkadiko",
+    "email": "philip@arkadiko.com"
```
this email does not exist, either philip@arkadiko.finance or niel@arkadiko.finance

nieldeckx commented:
@philipdesmedt 
- Implemented your feedback
- Updated contract so user can buy NFT with DIKO which is burned

TODO:
- Decide on distribution curve

philipdesmedt commented:
@nieldeckx LGTM - gonna merge this. We can do a separate PR to change the bonding curve
